security/symmetric/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "azure-iot-security-symmetric-key",
```

Comment:
>azure-iot-security-symmetric-key [](start = 11, length = 32)

I can't help but see very little value in this package except to provide symmetry with TPM and x509. I wonder if that's putting more hurdles in front of the customers than anything else (more packages to install etc)

Reply:
Disagree.  A think it is important to keep the experience the same between all methods.

---
In reply to: [231753550](https://github.com/Azure/azure-iot-sdk-node/pull/418#discussion_r231753550) [](ancestors = 231753550)

Reply:
Customer is fine with a separate package.

---
In reply to: [232118591](https://github.com/Azure/azure-iot-sdk-node/pull/418#discussion_r232118591) [](ancestors = 232118591,232014424,231753550)